### PR TITLE
2-customization: fixing issues

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: 2-customization/README.rst [hunk 1]
```diff
@@ -53,14 +52,13 @@ Customize
...
-Open with your favorite editor
-``/vagrant/2-customization/invenio-beauty/invenio_beauty/config.py``
+Open with your favorite editor the ``~/.virtualenvs/invenio/var/instance/invenio.cfg``
 
 1. Modify the logo
 ^^^^^^^^^^^^^^^^^^
 Let's make our theme beautiful by replacing the logo
 
-in the ``/vagrant/2-customization/invenio-beauty/invenio_beauty/config.py``
+in the ``~/.virtualenvs/invenio/var/instance/invenio.cfg`` add the following:
```
Comment (remileduc) [inline 2-customization/README.rst:61]: it may be better to do a `cdvirtualenv`?

Comment (drjova) [reply 2-customization/README.rst:61]: hmm, then we need each time to explain them ``cdvirtualenv var/instance/`` and then ``vi invenio.cfg`` :/

File: 2-customization/README.rst [hunk 2]
```diff
@@ -34,7 +34,6 @@ First go in the virtual machine:
 .. code-block:: console
 
     laptop> vagrant ssh web
-    vagrant> source .inveniorc
     vagrant> workon invenio
```
Comment (remileduc) [inline 2-customization/README.rst:39]: you may want to add the `-e` optoin to pip install as you add templates later

Comment (drjova) [reply 2-customization/README.rst:39]: we don't need it anymore, we are using the ``cfg`` directly :)

The templates would be already installed with ``pip install .`` and then we change only the config

**Pull request merged**